documentation/api/io.strimzi.api.kafka.model.CruiseControlSpec.adoc

```diff
@@ -147,6 +147,102 @@ spec:
 # ...
 ----
 
+To access the Cruise Control REST API with security is enabled, you can create API user.
+
+.Cruise Control configuration to add REST API users
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  cruiseControl:
+    apiUsers:
+      - name: my-user (1)
+        role: USER (2)
+# ...
+----
+1. The name of the API user.
+2. Valid API roles are VIEWER, USER, and ADMIN".
+For more information on valid API user roles see https://github.com/linkedin/cruise-control/wiki/Security#authorization
+
```

Comment:
```suggestion

```

Reply:
We have this link in two places. Maybe we can remove them and put the link in a note as below.